tooling/hibernate-gradle-plugin/src/main/java/org/hibernate/orm/tooling/gradle/EnhancementHelper.java

```diff
@@ -150,27 +185,16 @@ private static void writeOutEnhancedClass(byte[] enhancedBytecode, File file, Lo
 			logger.warn( "Problem preparing class file for writing out enhancements [" + file.getName() + "]" );
 		}
 
-		try {
-			FileOutputStream outputStream = new FileOutputStream( file, false );
-			try {
-				outputStream.write( enhancedBytecode );
-				outputStream.flush();
-			}
-			catch (IOException e) {
-				throw new GradleException( "Error writing to enhanced class [" + file.getName() + "] to file [" + file.getAbsolutePath() + "]", e );
-			}
-			finally {
-				try {
-					outputStream.close();
-				}
-				catch (IOException ignore) {
-				}
-			}
+		try ( FileOutputStream outputStream = new FileOutputStream( file, false ) ) {
+			outputStream.write( enhancedBytecode );
+			outputStream.flush();
 		}
 		catch (FileNotFoundException e) {
 			throw new GradleException( "Error opening class file for writing : " + file.getAbsolutePath(), e );
 		}
-
+		catch (IOException e) {
+			throw new GradleException( "Error writing to enhanced class [" + file.getName() + "] to file [" + file.getAbsolutePath() + "]", e );
```

Comment:
The above message is confusing to me. Seems the verbiage can be improved (maybe `Error writing enhanced result (class: [xxx], file: [xxx])`?)

Reply:
meh, there was just an extra `to`